notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,714 @@
+    "# it's run after the fork() and before  exec() to run the shell.\n",
+    "pro = subprocess.Popen([\"tritonserver --model-repository=/data/model_repository\"], stdout=subprocess.PIPE, \n",
+    "                       shell=True, preexec_fn=os.setsid) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install dependencies to run Triton Client from Triton Inference Server container\n",
+    "!pip3 install nvidia-pyindex\n",
+    "!pip3 install tritonclient[http] # Install just http client for this notebook demo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check client library can be imported\n",
+    "import numpy\n",
+    "import tritonclient.http as triton_http"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up HTTP client.\n",
+    "http_client = triton_http.InferenceServerClient(\n",
+    "    url='localhost:8000',\n",
+    "    verbose=False,\n",
+    "    concurrency=1\n",
+    ")\n",
+    "\n",
+    "# Set up Triton input and output objects for both HTTP and GRPC\n",
+    "triton_input_http = triton_http.InferInput(\n",
+    "    'input__0',\n",
+    "    (X_test.shape[0], X_test.shape[1]),\n",
+    "    'FP32'\n",
+    ")\n",
+    "triton_input_http.set_data_from_numpy(X_test, binary_data=True)\n",
+    "triton_output_http = triton_http.InferRequestedOutput(\n",
+    "    'output__0',\n",
+    "    binary_data=True\n",
+    ")\n",
+    "\n",
+    "# Submit inference requests \n",
+    "request_http = http_client.infer(\n",
+    "    'fil',\n",
+    "    model_version='1',\n",
+    "    inputs=[triton_input_http],\n",
+    "    outputs=[triton_output_http]\n",
+    ")\n",
+    "\n",
+    "# Get results as numpy arrays\n",
+    "result_http = request_http.as_numpy('output__0')\n",
+    "\n",
+    "# Check that we got the same accuracy as previously\n",
+    "predictions = [round(value) for value in result_http]\n",
+    "accuracy = accuracy_score(y_test, predictions)\n",
+    "print(\"Accuracy: %.2f%%\" % (accuracy * 100.0))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above test accuracy score of the model deployed in Triton using FIL backend matches with the one previously computed using XGBoost library's predict function."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
Let's finish off with some sort of conclusion that clarifies what all they should have gotten out of this notebook.